src/main/java/training/App.java

```diff
@@ -8,7 +8,66 @@ public String getGreeting() {
         return "Hello consortium world.";
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) { Rectangle r = new Rectangle(10, 4);
```

Comment:
Here you should call new Shapes().getAreaofAllshapes() similar way as new App().getGreeting() is being called.